src/pages/educational-institutes/[id].tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+import IndividualPage1 from '@components/IndividualPage1'
+import { useRouter } from 'next/router'
+
+const EducationalInstitutePage = () => {
+  const router = useRouter()
+  const { id } = router.query
+
+  if (typeof id === 'undefined') {
+    // Handle the case where the id is undefined
+    return <div>Id is undefined</div>
+  }
```

Comment:
I think we can check if id is truthy or falsy, and do all checking logic in the same return